tvb/recon/cli/core.py

```diff
@@ -31,8 +31,6 @@ def __new__(cls: type, name: str, bases: Tuple[type],
                 attrs: Dict[str, object]) -> type:
         for key, value in list(attrs.items()):
             if not key.startswith('_'):
-                value:
-                    str
```

Comment:
This is valid syntax for local variable type declaration in Python 3.6; I think I wrote that just to convince PyCharm that value was a string.  Still, I admit I have no idea why it's on two lines, and deleting these lines is fine.